### PR TITLE
Documentation: Badge, Button, Card, Image, Input

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

File: src/components/badge/badge.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+          </li>
+          <li class="site-nav-link">
+            <a href="/src/components/image/image.html">Image</a>
+          </li>
+          <li class="site-nav-link">
+            <a href="/src/components/input/input.html">Input</a>
+          </li>
+        </ul>
+      </nav>
+      <main class="site-content">
```
Comment (tarunsankhla) [inline src/components/badge/badge.html:57]: Can you add comments in your html code

**Review by m26unkwn — APPROVED** [review]
Everything looks good to me.

File: src/components/button/button.css [hunk 2]
```diff
@@ -60,3 +62,9 @@
 .btn-lg {
   font-size: 1.5rem;
 }
+
+.btn-float {
+  position: fixed;
+  bottom: 2rem;
+  right: 2rem;
+}
```
Comment (Debarshi95) [inline src/components/button/button.css:70]: Use descriptive classnames that matches the property. 
.btn-float feels something related to float. 
You can may be use  .btn-fixed,   .btn-absolute and so on.

File: src/components/badge/badge.html (hunk 1 above)
Comment (pankaj-prog) [reply src/components/badge/badge.html:57]: yeah i will keep that in mind

File: src/components/button/button.css (hunk 2 above)
Comment (pankaj-prog) [reply src/components/button/button.css:70]: okay

**Pull request merged**